.github/ISSUE_TEMPLATE/adoption_request.md

```diff
@@ -0,0 +1,72 @@
+---
+name: Adoption Request
+about: Submit your feature to the project
+title: 'Adoption Request'
+labels: 'adoption'
+assignees: ''
+
+---
+
+# Adoption Request
+
+_Thank you for wanting to contribute to the project! We are very happy to see the functionalities of the
+EDC being extended. Providing this open-source is a great opportunity for others with similar requirements
+and to avoid additional work._
+
+_For any details about the guidelines for submitting features, please take a look at the
+[contribution categories](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/contribution_categories.md)._
+
+
+## General information
+
+Please provide some information about your project or code contribution. 
+
+_If you choose to be referenced as a "friend", these will be added to the [known friends list](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/known_friends.md)._
+_If you choose to add your feature as a core EDC component, links to your current code and correlated issues,
+discussions, and pull requests are of great importance._
+
+| Title | Description | Maintainer | Links
+| :--- | :--- | :--- | :---
+| _My awesome project_ | _This is an example._ | _name of organization or GitHub user_ | _link to repository, homepage, discussion, etc._
+
+
+## Adoption level
+
+Next, please tell us what level of adoption you intend to provide.
```

Comment:
since we don't have mutually exclusive selection (RadioButton) I would add something like "pick only one". You never know :)